packages/chart/src/components/BarChart/components/BarChart.Horizontal.tsx

```diff
@@ -258,7 +219,7 @@ export const BarChartHorizontal = () => {
                             verticalAnchor='middle'
                             fontWeight={'normal'}
                           >
-                            {barLabel}
+                            {defaultBarLabel}
```

Comment:
I'm not seeing a definition for defaultBarLabel

Reply:
Fixed!